docs/source/tracking.rst

```diff
@@ -750,6 +833,18 @@ location to server's artifact store. This will be used as artifact location for
 experiments that do not specify one. Once you create an experiment, ``--default-artifact-root``
 is no longer relevant to that experiment.
 
+Starting a server with the ``--serve-artifacts`` option defined will enable proxy access for artifacts.
```

Comment:
Nit: Instead of using future-tense "will" throughout the docs here, can we use present-tense phrasing (not an English expert, these are probably inaccurate tense names):

```suggestion
Starting a server with the ``--serve-artifacts`` option defined enables proxy access for artifacts.
```

One more small phrasing suggestion:

```suggestion
Starting a server with the ``--serve-artifacts`` option defined enables proxied access for artifacts.
```

Reply:
great call-out. Modified it here and a few other places.